src/NServiceBus.Persistence.ServiceFabric.AcceptanceTests/SagaDataStorage/When_storing_saga_with_custom_saga_attribute.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace NServiceBus.Persistence.ServiceFabric.AcceptanceTests.SagaDataStorage
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using global::TestRunner;
+    using Microsoft.ServiceFabric.Data;
+    using Microsoft.ServiceFabric.Data.Collections;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_storing_saga_with_custom_saga_attribute : NServiceBusAcceptanceTest, INeed<IReliableStateManager>
+    {
+        const string CustomCollectionName = "custom-collection-name";
+        const string CustomEntityName = "custom-entity-name";
+
+        IReliableStateManager stateManager;
+
+        [Test]
+        public async Task Should_use_fullname_as_collection_name()
```

Comment:
Name is misleading. Full name is not used for the collection name, but a custom collection name.